modules/cli/src/main/scala/scala/cli/commands/shared/SharedOptions.scala

```diff
@@ -289,13 +295,20 @@ final case class SharedOptions(
     )
   }
 
+  lazy val scalacOptionsFromFiles: List[String] =
+    argsFiles.flatMap(argFile =>
+      os.read(os.Path(argFile.file, os.pwd)).split(System.lineSeparator())
```

Comment:
Should there be some special error handling here?

Reply:
I think it's fine.